llvm/test/Analysis/UniformityAnalysis/AMDGPU/intrinsics.ll

```diff
@@ -329,6 +329,7 @@ bb:
 
 declare <4 x float> @llvm.amdgcn.mfma.f32.16x16x32.f16(<8 x half>, <8 x half>, <4 x float>, i32 immarg, i32 immarg, i32 immarg)
 declare <16 x float> @llvm.amdgcn.mfma.f32.32x32x16.f16(<8 x half>, <8 x half>, <16 x float>, i32 immarg, i32 immarg, i32 immarg)
+declare <16 x float> @llvm.amdgcn.mfma.f32.32x32x16.bf16(<8 x bfloat>, <8 x bfloat>, <16 x float>, i32 immarg, i32 immarg, i32 immarg)
```

Comment:
I thought we no longer need declaration

Reply:
I don't like omitting them. It breaks autoupgrade on future intrinsic changes, and it's a handy reference for what the arguments actually are